vision/detect_pdf.rb

```diff
@@ -21,7 +21,7 @@ def detect_pdf_gcs gcs_source_uri:, gcs_destination_uri:, project_id:
   require "google/cloud/vision"
   require "google/cloud/storage"
 
-  vision = Google::Cloud::Vision.new
+  vision = Google::Cloud::Vision.new project: project_id
```

Comment:
IIRC, a project id is unnecessary as the Vision API doesn't support it in a request. Project id was accidentally added to the client library.

However, the project could be added to `Google::Cloud::Storage.new`.

Reply:
Should I remove the project as an argument all together? I was just basing the arguments off another sample

Reply:
Storage could still use it so I'd say leave it.